exceptions.hpp

```diff
@@ -0,0 +1,13 @@
+#include "exception"
```

Comment:
hmm... doesn't compile
```suggestion
#include <stdexcept>
```

Reply:
Done. Sorry for that - I need a CI to check that things, because since I used clang from mac i do not really need to add libs like <algorithm> / <memory> and so on and code is still compiling on my machine.